web3/utils/encoding.py

```diff
@@ -275,3 +278,34 @@ def to_4byte_hex(hex_or_str_or_bytes):
         )
     hex_str = encode_hex(byte_str)
     return pad_hex(hex_str, size_of_4bytes)
+
+
+class DynamicArrayPackedEncoder(BaseArrayEncoder):
+    is_dynamic = True
+
+    def encode(self, value):
+        encoded_elements = self.encode_elements(value)
+        encoded_value = encoded_elements
+
+        return encoded_value
+
+
+def encode_single_packed(_type, value):
```

Comment:
This will be replaced with the eth-abi packed encoder once we get https://github.com/ethereum/eth-abi/pull/89 for web3.py

Reply:
It's normally good to leave this type of information in an embedded comment linking to the issue that must be closed before the code can be updated.

Reply:
and it will have to survive longer anyway, until we can require eth-abi v2+ in web3 v5